web/themes/joinup/templates/rdf_entity/rdf-entity--asset-release--compact.html.twig

```diff
@@ -28,5 +35,8 @@
         <div class="timeline__listing-item">{{ 'No distributions yet.'|t }}</div>
       {% endif %}
     </div>
+    <div class="timeline__read-more">
+      <a href="{{ url }}">{{ 'Read more'|t }}</a>
```

Comment:
Please:
- change to "more"
- uppercase
- all alt/title attributes with title for a11y